runtime/src/main/java/com/flipkart/flux/resource/StateMachineResource.java

```diff
@@ -13,24 +13,26 @@
 
 package com.flipkart.flux.resource;
 
-import javax.ws.rs.POST;
-import javax.ws.rs.PUT;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
-
 import com.flipkart.flux.api.StateDefinition;
 import com.flipkart.flux.api.StateMachineDefinition;
 import com.flipkart.flux.dao.iface.StateMachinesDAO;
 import com.flipkart.flux.domain.State;
 import com.flipkart.flux.domain.StateMachine;
 import com.google.inject.Inject;
 
+import javax.inject.Named;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
 import java.util.HashSet;
 import java.util.Set;
 
 /**
  * @understands Exposes APIs for end users
  */
+@Path("/api")
```

Comment:
Need confirmation:
Are we not having /api/teams/{teamName}/fsm.... because these are internal APIs?
Also is the case that StateMachine is something we call a Workflow internally and that external user is not aware of?